cdc/scheduler/balancer.go

```diff
@@ -68,12 +80,7 @@ func (r *tableNumberBalancer) FindTarget(
 
 	captureWorkload := make(map[model.CaptureID]int)
 	for captureID := range captures {
-		captureWorkload[captureID] = 0
-	}
-
-	for captureID, tables := range tables.GetAllTablesGroupedByCaptures() {
```

Comment:
Does this performance issue exists in `FindVictims`?

Reply:
`FindVictims` is called one magnitude less frequently than `FindTarget`, so the complexity is acceptable. In system tests, `FindVictims` did not have any performance problem.

Reply:
We can add comment about complexity warning to `GetAllTablesGroupedByCaptures`, in case of abuse in the future.

Reply:
Yes, sure